bin/yugabyted

```diff
@@ -2283,5 +2481,140 @@ def retry_op(func, timeout=180):
     raise RuntimeError("Failed after retrying operation for {} secs.".format(
         now - start_time))
 
+
+class SetupEnvBasedCredentials(object):
+    def __init__(self):
+        self.ysql_user = os.environ.get('YSQL_USER')
+        self.ysql_password = os.environ.get('YSQL_PASSWORD')
+        self.ysql_db = os.environ.get('YSQL_DB')
+        self.ycql_user = os.environ.get('YCQL_USER')
+        self.ycql_password = os.environ.get('YCQL_PASSWORD')
+        self.ycql_keyspace = os.environ.get('YCQL_KEYSPACE')
+
+    def is_valid(self, var_name):
+        return True if var_name is not None and var_name else False
```

Comment:
you can just say `return var_name` here instead of this boolean expression

Reply:
The motto of `is_valid` is to verify the argument variable is `non-empty` and `not None`. And return a boolean value.
The suggested one returns the variable, but we need boolean value.

1. If a variable doesn't exist then `os.environ.get('VAR')` returns None.
2. If a variable is empty like 

```
export VAR=
os.environ.get('VAR')
```
then `VAR` value will be ''. And this can create some fail scenario.

**Scenario:**
If users set `export YSQL_PASSWORD=` then, they'll believe they're setting an empty password. In reality, because of the variable, ysqlsh works in enforced authentication mode, and ysqlsh doesn't support an empty password in enforced authentication mode.

Same case in other env variables.

Reply:
In new commit, `is_valid()` has removed.